src/plugins/dashboard/public/dashboard_container/component/grid/_dashboard_grid.scss

```diff
@@ -54,6 +57,7 @@
   width: 100% !important; /* 1 */
   top: 0 !important; /* 1 */
   left: 0 !important; /* 1 */
+  transform: translate(0, 0) !important; /* 1 */
```

Comment:
Need to translate the expanded item to the top left corner to negate any other positioning when expanding 👍